physics/dynamic_frame_body.hpp

```diff
@@ -85,6 +85,39 @@ DynamicFrame<InertialFrame, ThisFrame>::GeometricAcceleration(
   return gravitational_acceleration_at_point + fictitious_acceleration;
 }
 
+template<typename InertialFrame, typename ThisFrame>
+Vector<Acceleration, ThisFrame>
+DynamicFrame<InertialFrame, ThisFrame>::RotationFreeGeometricAccelerationAtRest(
```

Comment:
I would really prefer if we factored out this code by having an auxiliary function that is mostly a copy of `GeometricAcceleration` with 5 out parameters for the 5 components of the acceleration.  `GeometricAcceleration` would just call that function and sum the results; and `RotationFreeGeometricAccelerationAtRest` would call it with a null velocity (because "at rest") and drop the Euler acceleration.